api/test/pact/constants/work-allocation/tasks.spec.ts

```diff
@@ -8,7 +8,7 @@ const KILI_MUSO = {
   location: 'Taylor House',
   taskName: 'Apply for probate',
   dueDate: new Date(1604938789000),
-  assignee: toAssignee(CASEWORKERS.JOHN_SMITH).userName
+  //assignee: toAssignee(CASEWORKERS.JOHN_SMITH).userName
```

Comment:
We need at least _some_ tasks assigned. Please revert this.

Reply:
As discussed, will have some cases assigned but will leave Kili Muso unassigned